forgettable-frontend/src/Components/IconButton/IconButton.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable max-len */
+import React from 'react';
+import {Button} from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+
+function IconButton({btnText, onClick, includeIcon}) {
+  const prmry = getComputedStyle(document.documentElement).getPropertyValue('--prmry');
+  const txt3 = getComputedStyle(document.documentElement).getPropertyValue('--txt3');
+  const fontMedium = getComputedStyle(document.documentElement).getPropertyValue('--font-medium');
+  const textLarge = getComputedStyle(document.documentElement).getPropertyValue('--text-large');
+  const radiusBtn = getComputedStyle(document.documentElement).getPropertyValue('--radius-btn');
```

Comment:
This stuff is pretty clean! I should start doing this too - so ugly when it's inline